docs/ref/contrib/postgres/search.txt

```diff
@@ -118,6 +127,15 @@ Examples:
 See :ref:`postgresql-fts-search-configuration` for an explanation of the
 ``config`` parameter.
 
+.. versionchanged:: 3.1
+
+    Support for ``'websearch'`` search type and query expressions in
```

Comment:
Is this `versionchanged` section relevant to this PR?

Reply:
Ahh thanks for catching this @adamzap, there aren't other mentions about versions in this doc so I think we should remove all the versionchanged sections